### ISSUE TITLE
运行python eval.py --model_file <your path to wave.ckpt-done> --image_file img/test.jpg出错

### ISSUE THREAD

**Issue by 5663015** [hzy46/fast-neural-style-tensorflow#9]
![image](https://cloud.githubusercontent.com/assets/8531141/23492242/360e1f76-ff3e-11e6-946f-1054f5c97f31.png)
我是在win7下运行的，是系统问题还是缺少组件？望大神解答，谢谢~~~

**Comment by ghost** [comment]
你是不是python和tensorflow的版本比较新？我自己fork过一份code出来，然后要自己改了才能支持python 3 和最新的tesorflow 1.0

**Comment by 5663015** [comment]
@purlvin 
我python3.5.1，tf 0.12.作者用的好像是python2.7？要怎么改呀

**Comment by ghost** [comment]
@5663015 
我记不太清楚了， 那是我很早以前该过的，好似是padding的function的参数还是名字变了。。你可以搜一下的。或者你可以对比一下我改过的版本，不过那里面还有针对tensorflow 1.0.0的改动。。
https://github.com/purlvin/MachineLearning.git

**Comment by 5663015** [comment]
@purlvin 
好吧，谢谢啦，有问题再向您请教~~~

**Comment by ghost** [comment]
@5663015 客气了，互相学习~

**Comment by hzy46** [comment]
关于checkpoint读取出错

之前的代码是Tensorflow 0.11的，默认保存的checkpoint是Tensorflow V1格式。Tensorflow 更新到0.12后，就默认使用V2 格式的checkpoint，因此会出问题。commit 5309a2a 中已经注明write_version=tf.train.SaverDef.V1。更新到这个commit可以解决问题。

--------------------------------
About Error on Reading Checkpoint

In Tensorflow 0.11, the default format for checkpoints is "V1". After Tensorflow 0.12, the default format becomes "V2". So it will be an error when reading V1 formatted checkpoint. In commit 5309a2a, I use "write_version=tf.train.SaverDef.V1" to address this problem.

**Comment by mxl1990** [comment]
这里问题可以这样解决
tf.pad要求paddings参数要是tf.int32的，修改代码model.py文件中
 x_padded = tf.pad(x, [[0.0, 0.0], [kernel / 2.0, kernel / 2.0], [kernel / 2.0, kernel / 2.0], [0.0, 0.0]], mode=mode)
为
padding = tf.cast([[0.0, 0.0], [kernel / 2.0, kernel / 2.0], [kernel / 2.0, kernel / 2.0], [0.0, 0.0]], tf.int32)
x_padded = tf.pad(x, paddings=padding, mode=mode)
